packages/runtime/src/next/components/tests/prop-controllers/page-backgrounds-prop-controller.test.tsx

```diff
@@ -81,7 +81,7 @@ describe('Page', () => {
 
     await act(async () =>
       render(
-        <ReactRuntimeProvider runtime={runtime}>
+        <ReactRuntimeProvider runtime={runtime} previewMode={false}>
```

Comment:
A ton of identical prop controller test updates here and below, these will go away when we migrate prop controllers to the unified controls interface.